javaagent/docker-compose.yml

```diff
@@ -4,19 +4,24 @@ services:
     build: ./
     environment:
       OTEL_SERVICE_NAME: "agent-example-app"
-      OTEL_EXPORTER_OTLP_ENDPOINT: "http://collector:4317"
+      OTEL_EXPORTER_OTLP_ENDPOINT: "http://collector:4318"
```

Comment:
This is the default for 2.0 right....so do we like it to be explicit or can we just omit it?

Reply:
The endpoint defaults to `https://localhost:4318`, we need `http://collector:4318` 🙂